packages/tools/src/tools/annotation/splines/types/CardinalSplineProps.ts

```diff
@@ -0,0 +1,6 @@
+import { SplineProps } from './SplineProps';
+
+export type CardinalSplineProps = SplineProps & {
```

Comment:
Types should move to tools/types and maybe we create name space of SplineTypes then we can do SlineTypes.CardinalProps etc.

Reply:
All missing docs